### PR TITLE
Release 1.5.0: Merge development -> master

### PR DESCRIPTION
**Pull request by davidgrayston**
### Added
- `Yoti::Attribute#anchors` to retrieve a list of all anchors, including unknown types.
- `Yoti::Anchor#type` to get the type of anchor
- `Yoti::ActivityDetails#application_profile`
- `Yoti::ApplicationProfile`, which extends `Yoti::BaseProfile`
- `Yoti::ActivityDetails#receipt_id`
- `Yoti::ActivityDetails#timestamp`
- `Yoti::Configuration#api_endpoint`
- `X-Yoti-SDK-Version` request header

### Changed
- `Yoti::Profile` now extends `Yoti::BaseProfile`

### Fixed
- `Yoti::SignedTimeStamp#time_stamp` now has microsecond accuracy

### Deprecated
- `Yoti::AnchorProcessor#get_anchor_by_oid`
- `Yoti::AnchorProcessor#get_anchor_value_by_oid`
- `Yoti::AnchorProcessor#get_anchor_value_by_asn1_sequence`
- `Yoti::AnchorProcessor#get_anchor_value_by_asn1_data`
- `Yoti::AnchorProcessor#anchor_types`
- `Yoti::AnchorProcessor#get_next`
- `Yoti::ActivityDetails#user_profile`
- `Yoti::ActivityDetails#structured_postal_address`
- `Yoti::Protobuf#current_user`

### CODE REVIEW

File: lib/yoti/activity_details.rb [hunk 1]
```diff
@@ -1,69 +1,167 @@
...
+    # @return [ApplicationProfile]
+    #
+    def application_profile
+      Yoti::ApplicationProfile.new(@extended_application_profile)
     end
 
     protected
 
+    #
+    # Process the decrypted profile to into hash
```
Comment (echarrod) [inline lib/yoti/activity_details.rb]: `Process the decrypted profile into a hash`?

File: spec/yoti/client_spec.rb [hunk 2]
```diff
@@ -93,6 +126,30 @@
...
+
+      it 'contains the fetched application profile' do
+        expect(application_profile).not_to be_nil
+      end
+
+      it 'contains the application name' do
+        expect(application_profile.name.value).to eql('Node SDK Test')
+      end
+
+      it 'contains the application logo is nil' do
```
Comment (echarrod) [inline spec/yoti/client_spec.rb]:
```suggestion
      it 'has the application logo set to nil' do
```

File: lib/yoti/activity_details.rb (hunk 1 above)
Comment (davidgrayston) [reply lib/yoti/activity_details.rb]: It processes the profile into https://docs.ruby-lang.org/en/2.0.0/Hash.html - should we reword this?

Comment (davidgrayston) [reply lib/yoti/activity_details.rb]:
```suggestion
    # Process the decrypted profile to into key-value hash
```

Comment (echarrod) [reply lib/yoti/activity_details.rb]: 👍

**Comment by davidgrayston** [comment]
@echarrod last few changes in 5e43455...791bfb3
- Use local checked out SDK, to save us needing to update example project after each release
- Updated version for release

**Pull request merged**